ios/FluentUI/Core/SwiftUI+ViewModifiers.swift

```diff
@@ -31,4 +63,16 @@ extension View {
 
         return AnyView(self)
     }
+
+    /// Ensures that the modified View has equal height and width values based on the
+    /// maximum of either dimension considering its dynamic content size.
+    /// Minimum and maximum cap values need to be specified.
+    /// - Parameters:
+    ///   - minSize: Minimum heigh/width value of the view.
+    ///   - maxSize: Maximum heigh/width value of the view.
+    /// - Returns: Modified view in a square shaped format.
+    func squareShaped(minSize: CGFloat, maxSize: CGFloat) -> some View {
```

Comment:
Do we have plans to apply this to multiple views? If not, should it just be part of the HUD?

Reply:
Yeah! That's a good point. So far I believe the HUD is the only control with this requirement. So your suggestion makes a lot of sense even though the addition of this modifier will not make a significant size impact.
I can move it to the HUD folder when I send the PR for the new SwiftUI HUD control.
So far all of these new modifiers being introduced are not yet being used and the goal is just to make them easier to review.